app/src/main/res/layout/activity_username.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="utf-8"?>
+<merge xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
An Activity layout uses a `merge`? That's the first time I've seen it. Does it work? It's usually a CoordinatorLayout, for example.